source/unified-test-format/tests/valid-pass/entity-runCursorCommand.yml

```diff
@@ -0,0 +1,115 @@
+description: runCursorCommand
+schemaVersion: '1.3'
+createEntities:
+  - client:
+      id: &client client
+      useMultipleMongoses: false
+      observeEvents: [commandStartedEvent]
+  - database:
+      id: &db db
+      client: *client
+      databaseName: *db
+  - collection:
+      id: &collection collection
+      database: *db
+      collectionName: *collection
+initialData:
+  - collectionName: collection
+    databaseName: *db
+    documents: &documents
+      - { _id: 1, x: 11 }
+      - { _id: 2, x: 22 }
+      - { _id: 3, x: 33 }
+      - { _id: 4, x: 44 }
+      - { _id: 5, x: 55 }
+tests:
+  - description: creates a cursor and iterates results from getMores
```

Comment:
It's still not clear what causes the differences in the expected results between this case and the one below it.